lib/api/policy.go

```diff
@@ -93,6 +93,10 @@ type PolicySpec struct {
 	// 	deployment != "dev"
 	// 	! has(label_name)
 	Selector string `json:"selector" validate:"selector"`
+
+	// Indicates, if True, that the rules in this policy should be applied before any data plane
+	// connection tracking, and that packets allowed by these rules should not be tracked.
```

Comment:
Yup @caseydavenport is right and you can slap me for not adhering to this format - but we should do - i'll make sure there is an issue to fix up existing doc strings, but might as well start getting it right :-)